### PR TITLE
false: add page

### PR DESCRIPTION
**Pull request by sbrl**
Since @pxgamer  opened a PR for `true` (#2270), I decided to do one for `false`.

Part of #2213.

----

- [ ] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [ ] The page has 8 or fewer examples.

- [ ] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [ ] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/false.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# false
+
+> Does nothing, unsucessfully.
```
Comment (agnivade) [inline pages/linux/false.md]: I wonder if this line is helpful. Removing this doesn't change much.

Comment (pxgamer) [reply pages/linux/false.md]: I think keeping `true` (#2270) and `false` in the same format would be good.

Comment (sbrl) [reply pages/linux/false.md]: @pxgamer ...so remove this line/

Comment (agnivade) [reply pages/linux/false.md]: Yes. I think it makes sense to keep both pages in sync.

**Pull request merged**